packages/react-i18n/src/i18n.ts

```diff
@@ -31,6 +33,11 @@ export interface TranslateOptions {
   scope: RootTranslateOptions<any>['scope'];
 }
 
+// Used for currecies that don't use fractional units (eg. JPY)
+const DECIMAL_NOT_SUPPORTED = 'N/A';
+const DECIMAL_VALUE_FOR_CURRENCIES_WITHOUT_DECIMALS = '00';
+const DEFAULT_VALUE = '';
```

Comment:
I realize the name `DEFAULT_VALUE` is too generic here since it's defined in the `i18n.ts` file. Maybe you don't need that specific constant and can just return `''` in your method.